src/main/java/com/joss/conductor/mobile/ConductorConfig.java

```diff
@@ -502,6 +505,28 @@ public String getFullAppPath() {
         }
 
         Path path = Paths.get(appFile);
+        try {
+            if (platformName == Platform.ANDROID) {
+                    File dir = new File(appFile);
+                    File[] files = dir.listFiles((dir1, name) -> name.toLowerCase().endsWith(".apk"));
+                    if (files != null) {
+                        File apkName = files[0];
+                        Logger.warn("Found an APK at relative path" + apkName);
+                        return Paths.get(System.getProperty("user.dir"), apkName.toString()).normalize().toString();
+                    }
+            } else {
+                File dir = new File(appFile);
+                File[] files = dir.listFiles((dir1, name) -> name.toLowerCase().endsWith(".ipa"));
+                if (files != null) {
+                    File apkName = files[0];
+                    Logger.warn("Found an IPA at relative path" + apkName);
+                    return Paths.get(System.getProperty("user.dir"), apkName.toString()).normalize().toString();
+                }
+            }
+        }
+             catch (Exception e) {
+                Logger.error("Could not find IPA/APK at destination" + appFile);
```

Comment:
There are exceptions that could point to other issues other than an inability to find the package. I'd generalize this message to "Error attempting to find {appFile}: {e.message}" (however that's done properly in Java ;).  Also watch the formatting here, it got off somehow.

Reply:
Fixed the formatting and improved exception handling.